style.css

```diff
@@ -1,20 +1,117 @@
-*{
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-    height: 100vh;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
-.box{
-    background-color: blueviolet;
-    border: 1px solid black;
-    width: 200px;
-    height: 50vh;
-    flex: 1;
+
+@media only screen and (max-width: 768px) {
+  body {
+    height: 100vh;
+    width: 100%;
+    left: 0;
+    top: 0;
+    background-color: #fff;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 4vh 5vh;
+  }
+
+  .top-logo h2 {
+    font-family: 'Poppins', sans-serif;
+    font-size: 22px;
+    color: #6070ff;
+  }
+
+  .menu-icon {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    width: 30px;
+    height: 25px;
+  }
+
+  .menu-vector {
+    width: 30px;
+    height: 3px;
+    background-color: #6070ff;
+
+    /* border: 1px solid black; */
+  }
+
+  main {
+    display: flex;
+    background-color: rgb(243, 245, 247);
+    height: 100vh;
+    padding: 10vh 5vh;
+  }
+
+  .informative-top {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .top-heading h1 {
+    padding-right: 30%;
+    font-family: 'Poppins', sans-serif;
+    color: #172b4d;
+    font-weight: bolder;
+  }
+
+  .heading-description p {
+    font-family: 'Poppins', sans-serif;
+    font-size: 16px;
+    padding: 2vh 0;
+    color: #344563;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  .touch-links h3 a {
+    color: #7f8cff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    text-decoration: none;
+  }
+
+  .social-bar a img {
+    width: 19px;
+    height: 19px;
+  }
+
+  .social-bar h3 a {
+    padding: 0 5px;
+  }
+
+  .social-bar {
+    padding: 11px 0;
+  }
```

Comment:
- [ ] Please make sure the margin between the social media icon matches that of the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=RfqrWNFQxQ6al11h-0)

**_Your design_**

<img width="127" alt="Screenshot 2023-02-16 at 11 22 44 AM" src="https://user-images.githubusercontent.com/44798044/219298196-60d81fb4-0f70-4258-84d9-a5f4900b24dd.png">

**_Figma design_**

<img width="175" alt="Screenshot 2023-02-16 at 12 36 03 PM" src="https://user-images.githubusercontent.com/44798044/219298602-a5070ce0-b681-4c16-bd47-04cf4b134007.png">